keel-sql/src/main/resources/db/databaseChangeLog.yml

```diff
@@ -143,3 +143,6 @@ databaseChangeLog:
   - include:
       file: changelog/20200521-event-uid-timestamp-index.yml
       relativeToChangelogFile: true
+  - include:
+      file: changelog/20200521-add-event-pk.yml
```

Comment:
Is it OK that `20200521-add-event-pk.yml` precedes `20200521-event-uid-timestamp-index.yml`in lexicographical order? I recall this biting us before.

Reply:
I don't _think_ they should conflict (and the latter is rolled out in all our environments already)